desc/objectives/linear_objectives.py

```diff
@@ -2634,6 +2634,11 @@ class FixCoilCurrent(FixParameters):
         Whether target and bounds should be normalized before comparing to computed
         values. If `normalize` is `True` and the target is in physical units,
         this should also be set to True.
+    indices : ndarray or bool, optional
```

Comment:
could this instead be more like the optimizable way where it is a pytree? Or could it be both? to allow for fixing currents when things are nested